ui/src/app/components/demographic-survey-v2.tsx

```diff
@@ -179,6 +182,12 @@ const validateDemographicSurvey = (demographicSurvey: DemographicSurveyV2) => {
         message: "^ Difficulty doing errands can't be blank",
       },
     },
+    disabilityOtherText: {
+      length: {
+        maximum: 1000,
```

Comment:
how did you determine that 1000 was the correct length here?

Would you mind making a constant for this value, and other uses of 1000 in this file?

Reply:
Whoops, that should be 200, because that is the DB limit for that field. I will update the value and set it to a constant. Thanks for drawing my attention to this.